assets/js/blocks/mini-cart/style.scss

```
@@ -42,23 +42,36 @@
 .wc-block-mini-cart__drawer {
 	font-size: 1rem;
 
+	.components-modal__content .components-modal__header .components-button {
+		top: 50%;
+		padding: unset;
+		margin: unset;
+	}
+
+	.wc-block-components-notices {
+		padding: 0 16px;
+		margin-bottom: unset;
+	}
+
 	.components-modal__content {
 		padding: 0;
 		position: relative;
 	}
 
 	.components-modal__header {
-		position: absolute;
-		top: $gap-largest;
-		right: $gap;
+		position: relative;
+		height: calc( $gap-largest + $gap );
+
```

And here we recover the previous styles:
```suggestion
		position: absolute;
		top: $gap-largest;
		right: $gap;
```